packages/c/csfml/xmake.lua

```diff
@@ -0,0 +1,29 @@
+package("csfml")
+    set_homepage("https://www.sfml-dev.org/")
+    set_description("CSFML - Official C binding for SFML")
+    set_license("zlib")
+
+    add_urls("https://github.com/SFML/CSFML/archive/refs/tags/$(version).tar.gz")
+    add_versions("2.6.1", "f3f3980f6b5cad85b40e3130c10a2ffaaa9e36de5f756afd4aacaed98a7a9b7b")
+
+    add_deps("cmake")
+    add_deps("sfml =2.6.1")
```

Comment:
This should be done in a on_load callback with the package version to support multiple versions